packages/core/src/telemetry/detached-flush.test.ts

```diff
@@ -0,0 +1,37 @@
+import child_process from "node:child_process";
+import { rmSync, writeFileSync } from "node:fs";
+import { tmpdir } from "node:os";
+import path from "node:path";
+import process from "node:process";
+import { expect, test } from "vitest";
+
+test("detached flush script should run without error", async ({ common }) => {
```

Comment:
Moved this test to its own file, because it's really testing the detached-flush script on its own, not as it is used by the telemetry service.